packages/discovery-provider/src/api/v1/metrics.py

```diff
@@ -68,6 +68,12 @@
     "month": ["day", "week"],
     "all_time": ["month", "week"],
 }
+valid_bucket_sizes_with_year = {
```

Comment:
is it worth being dry and referencing the values in the top object to prevent divergence?